packages/theme/src/components/ThemeProvider.tsx

```diff
@@ -13,7 +13,8 @@ import {
 } from "../types";
 
 export interface ThemeState {
-  mode: ThemeMode;
+  fullMode: ThemeMode;
+  mode: Omit<ThemeMode, "pref">;
```

Comment:
Could you please add some comments to this interface?  It's not obvious to me what fullMode vs mode vs key is.

Reply:
Yeah, that is a good idea! Added in [add documentation to different mode values](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9217/commits/c979e407889d7c6008359cff5c149ebe0bf01c4c). Hopefully that makes sense 😅